cmd/cloud.go

```diff
@@ -65,11 +65,13 @@ This will execute the test on the Load Impact cloud service. Use "k6 login cloud
         k6 cloud script.js`[1:],
 	Args: exactArgsWithMsg(1, "arg should either be \"-\", if reading script from stdin, or a path to a script file"),
 	RunE: func(cmd *cobra.Command, args []string) error {
+		logger := logrus.StandardLogger()
+
 		//TODO: disable in quiet mode?
 		_, _ = BannerColor.Fprintf(stdout, "\n%s\n\n", consts.Banner)
 
 		progressBar := pb.New(pb.WithConstLeft(" Init"))
-		printBar(progressBar, "Parsing script")
+		printBar(progressBar, "Parsing script", logger)
```

Comment:
Yeah, repeating this passing of `logger` could definitely be improved... What's more, `printBar` doesn't actually need the logger, it just has to pass it to the `progressBar.Render()` anyway.. So having a `WithLogger()` configurator (for lack of a better word for that pattern of functions :sweat_smile: ) would allow you to just specify it once:
```go
progressBar := pb.New(
    pb.WithConstLeft(" Init"),
    pb.WithLogger(logger),
)
```

Reply:
Good idea, not sure how I didn't consider it. :)

See 84bf065c.